modules/cluster-logging-rn-5.4.5.adoc

```diff
@@ -0,0 +1,22 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-5"]
+= Logging 5.5.1
+This release includes link:https://access.redhat.com/errata/RHSA-2022:6183[RHSA-2022:6183-OpenShift Logging Bug Fix Release 5.4.5].
+
+[id="openshift-logging-5-4-5-bug-fixes"]
+== Bug fixes
+* Before this update, the Operator did not ensure that the pod was ready, which caused the cluster to reach an inoperable state during a cluster restart. With this update, the Operator marks new pods as ready before continuing to a new pod during a restart, which resolves the issue. (link:https://issues.redhat.com/browse/LOG-2881[LOG-2881])
+
+* Before this update, the addition of multi-line error detection caused internal routing to change and forward records to the wrong destination. With this update, the internal routing is correct. (link:https://issues.redhat.com/browse/LOG-2946[LOG-2946])
+
+[id="openshift-logging-5-4-5-cves"]
+== CVEs
+* link:https://access.redhat.com/security/cve/CVE-2022-1292[CVE-2022-1292]
+* link:https://access.redhat.com/security/cve/CVE-2022-1586[CVE-2022-1586]
+* link:https://access.redhat.com/security/cve/CVE-2022-1785[CVE-2022-1785]
+* link:https://access.redhat.com/security/cve/CVE-2022-1897[CVE-2022-1897]
+* link:https://access.redhat.com/security/cve/CVE-2022-1927[CVE-2022-1927]
+* link:https://access.redhat.com/security/cve/CVE-2022-2068[CVE-2022-2068]
+* link:https://access.redhat.com/security/cve/CVE-2022-2097[CVE-2022-2097]
+* link:https://access.redhat.com/security/cve/CVE-2022-30631[CVE-2022-30631]
```

Comment:
@libander we have 2 new CVE bugs added in to 5.4.5: https://issues.redhat.com/browse/LOG-3010 https://issues.redhat.com/browse/LOG-3011

Reply:
And I have a question here: we don't have fix for CVE-2022-1292 and other CVEs, do we still need to add them in the release notes?

Reply:
JIRA issues associated with CVEs don't need release notes, and the CVEs included in the errata may not be changed by docs or engineering. Resolving.